app/presenters/scope_filters_presenter.rb

```diff
@@ -1,29 +1,42 @@
 require 'gds_api/support_api'
 
 class ScopeFiltersPresenter
-  attr_reader :path, :organisation_slug
+  attr_reader :paths, :organisation_slug, :paths_for_api
 
-  def initialize(path: nil, organisation_slug: nil)
-    @path = normalize_path(path)
+  def initialize(paths: nil, organisation_slug: nil)
+    @paths = normalize_paths(paths)
```

Comment:
I find it sort of surprising that `normalize_paths` returns a string joined by commas, a return value that we store on `@paths`, but when you look into the implementation `normalize_paths` has also sets `@paths_for_api` to the array version of that string.  I think it would be clearer if either both assignments happened inside the method, or both happened outside the method.

Reply:
Following on from my comment on `done_page?` maybe we need something like:

```ruby
def initialize(paths: nil, ...)
  @some_internal_var_name_for_the_array_of_paths = normalize_paths(paths)
  ...
end

def paths
  some_internal_var_name_for_the_array_of_paths.join(', ')
end

def paths_for_api
  some_internal_var_name_for_the_array_of_paths
end

private
attr_reader :some_internal_var_name_for_the_array_of_paths
```

?

Reply:
I've used `parsed_path` in place of `:some_internal_var_name_for_the_array_of_paths`.